File: go/cq/example_test.go
package main

import (
	"testing"
)

func TestHelloCypher(t *testing.T) {
	res := HelloCypher()
	if res[0] != "JnBrymn" || res[1] != "technige" {
		t.Fatal("doesn't match!", res)
	}
}

File: python/py2neo/example_test.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import example
import unittest

class TestExample(unittest.TestCase):
    def test_hello_cypher(self):
        friends = example.hello_cypher()
        self.assertEqual(friends, ["JnBrymn", "technige"])

if __name__ == '__main__':
    unittest.main()
